app/src/main/java/com/futurice/freesound/feature/common/ui/adapter/DiffUtilCallback.java

```diff
@@ -0,0 +1,52 @@
+package com.futurice.freesound.feature.common.ui.adapter;
+
+import com.futurice.freesound.feature.common.DisplayableItem;
+
+import android.support.annotation.NonNull;
+import android.support.v7.util.DiffUtil;
+
+import java.util.List;
+
+final class DiffUtilCallback extends DiffUtil.Callback {
```

Comment:
I can't figure out how to import the copyright xml config files. I imported the style xml but that is not enough, right?